ribs/archives/_archive_base.py

```diff
@@ -160,9 +160,13 @@ def __init__(self,
                                       dtype=self.dtype)
         self._metadata_arr = np.empty(self._cells, dtype=object)
 
-        # For CMA-MAE
-        self._learning_rate = learning_rate
-        self._threshold_min = threshold_min
+        # threshold min can only be -np.inf if the learning rate is 1.0
+        if learning_rate != 1.0 and threshold_min == -np.inf:
```

Comment:
Comment on CMA-ME behavior in docstring

Reply:
Where should we put this, docstring for `archive_base`? for `compute_threshold_update`?